src/styles/main.scss

```diff
@@ -1,3 +1,63 @@
 body {
   margin: 0;
 }
+
+$column-size: 120px;
+$gap-size: 1px;
+$month-length: 28;
+$starts-day:
+  0 "mon", 1 "tue"
+  2 "wed", 3 "thu",
+  4 "fri", 5 "sat",
+  6 "sun";
+
+.calendar {
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: #{$column-size * 7};
```

Comment:
What is 7? Try to avoid magic numbers. Also, make sure that this width includes gaps because now there are only 6 days in a week
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/0d1e0ef9-c930-46ea-b186-e42881ffdd77)